### PR TITLE
Update build scripts to fix small errors

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

**Comment by wtgodbe** [comment]
@maririos PTAL

File: clean.cmd [hunk 1]
```diff
@@ -3,22 +3,94 @@ setlocal EnableDelayedExpansion
...
+
+:Loop
+if [%1]==[] goto Begin
+
 if /I [%1] == [/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
+if /I [%1] == [/p] (
+    set packages=true
+    set thisArgs=!thisArgs!%1
```
Comment (maririos) [inline clean.cmd]: This is not used. You could just delete it

File: clean.cmd [hunk 2]
```diff
@@ -3,22 +3,94 @@ setlocal EnableDelayedExpansion
...
 
 exit /b 0
 
 :Usage
 echo.
 echo Repository cleaning script.
-echo No option parameters.
+echo Options:
+echo     /b     - Cleans the bin directory
+echo     /p     - Cleans the packages directory
```
Comment (maririos) [inline clean.cmd]: it could be important to delete the cache too.

File: clean.cmd [hunk 3]
```diff
@@ -3,22 +3,94 @@ setlocal EnableDelayedExpansion
...
+echo     /t     - Cleans the tools directory
+echo.
+echo If no option is specified then clean.cmd /b /p /t is implied.
```
Comment (maririos) [inline clean.cmd]: what if you use `clean /all` ?

File: clean.cmd [hunk 4]
```diff
@@ -3,22 +3,94 @@ setlocal EnableDelayedExpansion
...
+if [%packages%] == [true] (
+	if exist "%__ProjectDir%\packages" (
+		echo Deleting packages directory
+		rd /s /q "%__ProjectDir%\packages"
+		if NOT [!ERRORLEVEL!]==[0] (
+  			echo ERROR: An error occurred while deleting the packages directory - error code is !ERRORLEVEL!
+  			exit /b 1
+  		)
+  	)
+)
```
Comment (maririos) [inline clean.cmd]: Maybe you could add a message here for the user saying that the clean was successful.

**Comment by maririos** [comment]
Apart from my comments, LTGM

**Pull request merged**